### PR TITLE
Update README.md adding the example of usage of -p,--parameter <param-name param-value>

### PR DESCRIPTION
**Pull request by julian-miras**

### CODE REVIEW

**Review by machaval — APPROVED** [review]

**Comment by machaval** [comment]
The PR is blocked because the commit is not signed. :( . I've created a new PR with just copying this. #55  @julian-miras either feel free to aprove that PR or please create a new one with your commits signed

File: README.md [hunk 1]
```diff
@@ -287,6 +287,21 @@ curl https://jsonplaceholder.typicode.com/posts/1 | dw "output application/json
 }
 ```
 
+### Using parameters with the map `params`
```
Comment (afelisatti) [inline README.md:290]:
```suggestion
### Using Parameters
```

File: README.md [hunk 2]
```diff
@@ -287,6 +287,21 @@ curl https://jsonplaceholder.typicode.com/posts/1 | dw "output application/json
...
+
+Using the internal map `params`, we can access to an injected parameter in the command line with the -p option
```
Comment (afelisatti) [inline README.md:292]:
```suggestion
Using the internal map `params`, we can access injected parameters in the command line with the `-p` option:
```

File: README.md [hunk 3]
```diff
@@ -287,6 +287,21 @@ curl https://jsonplaceholder.typicode.com/posts/1 | dw "output application/json
...
+
+```
+dw -p myName Julian "output applcation/json --- { name : params.myName }"
```
Comment (afelisatti) [inline README.md:295]:
```suggestion
dw -p myName Julian "output json --- { name : params.myName }"
```

**Review by afelisatti — CHANGES_REQUESTED** [review]
Thanks for your contribution, @julian-miras!!! I just added some typing suggestions and simplified the example (turns out we'd been using the more verbose syntax in all examples here, we'll need to change those later).

**Comment by julian-miras** [comment]
Thanks @machaval ! I knew I was something wrong... good learning for my first PR! I will install the GPG keys to sign my commits going forward, and I will use your PR instead of creating a new one. @afelisatti , I will incorporate your suggestions in the #55 so we will be good to go. I will close this one to clean it up. Thanks!

**Pull request closed without merging**